.github/workflows/node.js.yml

```diff
@@ -0,0 +1,95 @@
+name: Node.js CI
```

Comment:
let's update this name to be something that more relates to what this workflow is doing

potential options:
- Build
- Build & Deploy
- Build knowgod.com
- (something else that makes sense)